python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -755,6 +884,15 @@ def process_edge_data(process_confs, node_id_map, num_processes):
 
     return edges, edge_data
 
+def verify_confs(confs):
+    """ Verify the configuration of the input data.
+    """
+    ntypes = {conf['node_type'] for conf in confs["node"]}
+    etypes = [conf['relation'] for conf in confs["edge"]]
+    for src_type, _, dst_type in etypes:
+        assert src_type in ntypes, f"source node type {src_type} does not exist."
```

Comment:
Can we also tell user how to fix the problem or what is the root cause of the problem? For example, they do not add src_node_type in the config.